stock_picking_line_sequence/models/stock_move.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2017 Camptocamp SA - Damien Crier, Alexandre Fayolle
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import api, fields, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    # re-defines the field to change the default
+    sequence = fields.Integer("HiddenSequence", default=9999)
+
+    # displays sequence on the stock moves
+    sequence2 = fields.Integer(
```

Comment:
Seems now is a good time to change it to visible_sequence (Line Number) for naming consistency with other *_line_sequence modules.

Reply:
Thanks, done!